openedx/features/badging/handlers.py

```diff
@@ -26,3 +31,18 @@ def delete_badge_info_from_nodebb(sender, instance, **kwargs):
         'id': instance.id
     }
     task_delete_badge_info_from_nodebb.delay(badge_data)
+
+
+@receiver(startup.perform_type_registrations)
+def register_notification_types(sender, **kwargs):  # pylint: disable=unused-argument
```

Comment:
edx-notifications repo register all receives using import `from edx_notifications.signals import perform_type_registrations` like this

```
@receiver(perform_type_registrations)
def register_notification_types(sender, **kwargs):
```

Lets, please update the code
Also remove import `from edx_notifications import startup`

Reply:
done